tncc.py

```diff
@@ -564,13 +571,34 @@ def get_cookie(self, dspreauth=None, dssignin=None):
         post_data = ''.join([ '%s=%s;' % (k, v) for k, v in post_attrs.iteritems()])
         self.r = self.br.open('https://' + self.vpn_host + self.path + 'hc/tnchcupdate.cgi', post_data)
 
+        # set the inital host checker cookie to the first DSPREAUTH cookie
+        self.set_cookie('DSPREAUTH_HC', self.find_cookie('DSPREAUTH').value)
+
+        # parse the response to retrieve the periodic host checking interval
+        response = self.parse_response()
+
+        # try to get the interval from the response, if not fallback with except
+        try:
+            self.hc_interval = int(response['msg'].split("interval=")[1].split("SESSION")[0])
+        except:
+            self.hc_interval = 10
```

Comment:
It would be a edge case situation/possibly a TNCC protocol change if we cannot parse `interval`, so I propose logging (at debug level) a full `response` and fact of default fallback (at info level)

Reply:
Also, not every "Host Checker" instance is configured to do periodic checks. With this fallback, it would diverge from PS client implementation with unknown consequences (best case, somehow registering in VPN server logs as out-of-band HC responses; worst case - ?)